home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,56 @@
+# Local Library
+
+You can import your own files/objects from your PC into iVP by using the "Import" option, found under the "File" menu at the top of the application.  
+
+![](../../../.gitbook/assets/3d-import-file.png)
+ 
+
+When importing your file, you have the option to change its parameters, as well as the option to add it to your library, which then will show up at the bottom of the default library on the left side of the application.
+
+![](../../../.gitbook/assets/3d-import-addtolibrary.png)
+
+
+### iVP Library  
+
+Once you have added an item to your library, the folder "iVP Library" will appear at the bottom of the library panel.
+Every item that you import, and chose to add to your library, can be found inside the "iVP Library" folder.  
```

Comment:
This sentence does not really make sense grammatically and can also be simplified.

Reply:
Please also explain that a folder will be created on the hard drive where the files are copied to and where to find it by default.